nucypher/config/constants.py

```diff
@@ -44,10 +44,9 @@
 SeednodeMetadata = namedtuple('seednode', ['checksum_address', 'rest_host', 'rest_port'])
 SEEDNODES = tuple()
 
-
 # Sentry
-NUCYPHER_SENTRY_PUBLIC_KEY = "d8af7c4d692e4692a455328a280d845e"
-NUCYPHER_SENTRY_USER_ID = '1310685'
+NUCYPHER_SENTRY_PUBLIC_KEY = "fd21c3edda324065a34d3f334dddf1f0"
+NUCYPHER_SENTRY_USER_ID = '1480080'
```

Comment:
and this should be
```suggestion
NUCYPHER_SENTRY_USER_ID = os.getenv("NUCYPHER_SENTRY_USER_ID") or '1480080'
```

Reply:
Yeah - perhaps we need an Issue for standardizing where and how to define env vars - at the moment this check is done here: <https://github.com/nucypher/nucypher/pull/1069/files#diff-1ef88d57d518843bad3ce193d158843dR48>

Reply:
See related #838